modules/ROOT/pages/prereq-aws-terraform.adoc

```diff
@@ -122,7 +123,7 @@ After the provisioner runs successfully, it displays information about your envi
 You can have your own shell scripts to run on the provisioned instances before and/or after the PCE provisioner scripts. To do so, place your shell scripts with .sh extension inside a folder named `pre-user-data` and/or `post-user-data`. Include the following volume mounts on the docker run command:
 
 ----
-docker run --rm -v $(pwd)/pre-user-data:/usr/local/bin/provisioner/terraform/external/pre-user-data -v $(pwd)/post-user-data:/usr/local/bin/provisioner/terraform/external/post-user-data --env-file pce.env artifacts.msap.io/mulesoft/core-paas-private-cloud-provisioner:v3.0.17 cluster-provision
+docker run --rm -v $(pwd)/pre-user-data:/usr/local/bin/provisioner/terraform/external/pre-user-data -v $(pwd)/post-user-data:/usr/local/bin/provisioner/terraform/external/post-user-data --env-file pce.env artifacts.msap.io/mulesoft/core-paas-private-cloud-provisioner:v3.1.0 cluster-provision
```

Comment:
Double-checking: you want to change from 0.17 to 1.0 (rather than the change to 1.1 change in previous similarly changed texts)?

Reply:
I'm going to leave these like this for now. There's a separate ticket to update these. Thanks